### PR TITLE
remove undefined reponse from controller

### LINKED ISSUE #613: "undefined" shown with error message displayed on password reset view
Please see the error message in the screenshot. The response from the backend is correct. 
screen shot 2017-01-31 at 2 01 16 am

### PR DESCRIPTION
**Pull request by anweshknayak**
fixes #613 
fixundef

### CODE REVIEW

**Comment by anweshknayak** [comment]
@aka-jain please review

File: frontend/src/js/controllers/authCtrl.js [hunk 1]
```diff
@@ -257,9 +257,9 @@
                             if (token_valid) {
                                 vm.FormError = "this link has been already used or expired.";
                             } else if (password1_valid) {
-                                vm.FormError = response.data.new_password1[0] + " " + response.data.new_password1[1];
```
Comment (aka-jain) [inline frontend/src/js/controllers/authCtrl.js]: @spyshiv Please check this.

Comment (spyshiv) [inline frontend/src/js/controllers/authCtrl.js]: @anweshknayak : There are some cases when error response is something like this
```
new_password1 {
    0: "error 1",
    1: "error 2" 
}
```
So solve it by applying loop method till the length of `new_password1`

**Comment by anweshknayak** [comment]
@spyshiv @aka-jain review pls

**Comment by spyshiv** [comment]
@anweshknayak : Great :+1: 
LGTM. Merging this.

**Pull request merged**